static/app/components/events/eventReplay/replayClipPreviewPlayer.tsx

```diff
@@ -1,168 +1,88 @@
-import {useEffect} from 'react';
 import styled from '@emotion/styled';
 
 import NegativeSpaceContainer from 'sentry/components/container/negativeSpaceContainer';
-import {Alert} from 'sentry/components/core/alert';
 import type {LinkButtonProps} from 'sentry/components/core/button';
-import {
-  REPLAY_LOADING_HEIGHT,
-  REPLAY_LOADING_HEIGHT_LARGE,
-} from 'sentry/components/events/eventReplay/constants';
+import {REPLAY_LOADING_HEIGHT} from 'sentry/components/events/eventReplay/constants';
 import ReplayPreviewPlayer from 'sentry/components/events/eventReplay/replayPreviewPlayer';
 import {StaticReplayPreview} from 'sentry/components/events/eventReplay/staticReplayPreview';
 import LoadingIndicator from 'sentry/components/loadingIndicator';
 import ArchivedReplayAlert from 'sentry/components/replays/alerts/archivedReplayAlert';
-import MissingReplayAlert from 'sentry/components/replays/alerts/missingReplayAlert';
+import ReplayLoadingState from 'sentry/components/replays/player/replayLoadingState';
 import ReplayProcessingError from 'sentry/components/replays/replayProcessingError';
 import {t} from 'sentry/locale';
-import {trackAnalytics} from 'sentry/utils/analytics';
-import type {TabKey} from 'sentry/utils/replays/hooks/useActiveReplayTab';
 import type useLoadReplayReader from 'sentry/utils/replays/hooks/useLoadReplayReader';
-import type RequestError from 'sentry/utils/requestError/requestError';
-import useRouteAnalyticsParams from 'sentry/utils/routeAnalytics/useRouteAnalyticsParams';
-import useOrganization from 'sentry/utils/useOrganization';
+import useLogEventReplayStatus from 'sentry/utils/replays/hooks/useLogEventReplayStatus';
 import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
-import type {ReplayRecord} from 'sentry/views/replays/types';
 
-interface ReplayClipPreviewPlayerProps {
+interface Props {
   analyticsContext: string;
-  orgSlug: string;
+  fullReplayButtonProps: Partial<Omit<LinkButtonProps, 'external'>>;
   replayReaderResult: ReturnType<typeof useLoadReplayReader>;
-  focusTab?: TabKey;
-  fullReplayButtonProps?: Partial<Omit<LinkButtonProps, 'external'>>;
-  handleBackClick?: () => void;
-  handleForwardClick?: () => void;
-  isLarge?: boolean;
-  onClickNextReplay?: () => void;
-  overlayContent?: React.ReactNode;
-  showNextAndPrevious?: boolean;
-}
```

Comment:
The original ReplayClipPreviewPlayer had basically two sets of optional params.

Now we have just the one set, including `isLarge:false` and none of the click stuff.